Dockerfile

```diff
@@ -2,4 +2,4 @@
 FROM cerner/terra-node
 
 # run the server
-CMD ["npm", "run", "start"]
+CMD ["npm", "run", "start", "--", "--port", "8080"]
```

Comment:
This is temporary until we get a clean release right?

Reply:
Yes should be. @ryanthemanuel When do we push the 'latest' tagged docker image?

Reply:
Travis ci does it. https://github.com/cerner/terra-toolkit/blob/master/.travis.yml#L58

Reply:
Though, i'd be remiss if i didn't point out that if we were testing against this version and not the previously released version, this issue wouldn't have been released.

Reply:
Also i'm pretty sure my previous comment was wrong. Please disregard. :)